sass/cd/cd-header/_cd-user-menu.scss

```diff
@@ -67,6 +67,58 @@
   }
 }
 
+// Placeholder menu item with + icon, based on placement.
+.logged-in .cd-user-menu > li:first-child {
+  a {
+    padding: 0 12px 0 26px;
+
+    @include tablet {
+      padding: 0 20px 0 34px;
+    }
+  }
+  &:before {
+    position: absolute;
+    content: "";
+    height: $cd-global-header-height;
+    width: 14px;
+    left: 7px;
+    background-image: url(../img/icons/plus.svg);
```

Comment:
On my local this creates a plus icon next to "My Account" — maybe it's better to create individual extendables out of the icons and apply them to Drupal-specific selectors as needed.

<img width="487" alt="cd-31-my-account-plus" src="https://user-images.githubusercontent.com/254753/44458461-a13dda00-a606-11e8-99d6-88bb398f36af.png">

Reply:
Yep, that's correct is so far is it's what I intended. I thought this was a good way to demonstrate the additional icon since that item will likely be "+ add content" instead of "My account". Related to comment below.